internal/commands/parameters_test.go

```diff
@@ -51,38 +52,90 @@ func TestWithCommandLineParameters(t *testing.T) {
 	assert.Assert(t, cmp.DeepEqual(actual, expected))
 }
 
+type bundleOperator func(*bundle.Bundle)
+
+func prepareBundleWithParameters(b *bundle.Bundle) {
+	if b.Parameters != nil && len(b.Parameters.Fields) > 0 {
+		return
+	}
+	b.Parameters = &bundle.ParametersDefinition{
+		Fields:   map[string]bundle.ParameterDefinition{},
+		Required: []string{},
+	}
+	b.Definitions = definition.Definitions{}
+}
+
+func withParameter(name, typ string) bundleOperator {
+	return func(b *bundle.Bundle) {
+		prepareBundleWithParameters(b)
+		b.Parameters.Fields[name] = bundle.ParameterDefinition{
+			Definition: name,
+		}
+		b.Definitions[name] = &definition.Schema{
+			Type: typ,
+		}
+	}
+}
+
+func withParameterAndDefault(name, typ string, def interface{}) bundleOperator {
+	return func(b *bundle.Bundle) {
+		prepareBundleWithParameters(b)
+		b.Parameters.Fields[name] = bundle.ParameterDefinition{
+			Definition: name,
+		}
+		b.Definitions[name] = &definition.Schema{
+			Type:    typ,
+			Default: def,
+		}
+	}
+}
+
+func withParameterAndValues(name, typ string, allowedValues []interface{}) bundleOperator {
+	return func(b *bundle.Bundle) {
+		prepareBundleWithParameters(b)
+		b.Parameters.Fields[name] = bundle.ParameterDefinition{
+			Definition: name,
+		}
+		b.Definitions[name] = &definition.Schema{
+			Type: typ,
+			Enum: allowedValues,
+		}
+	}
+}
+
+func mockBundle(ops ...bundleOperator) *bundle.Bundle {
```

Comment:
Just nit picking, but what about renaming it to `prepareBundle`? I don't think it's a mock as it returns a real bundle.

Reply:
Well, it's not a real bundle since it only contains parameters and definitions but OK.